hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/RpcConnection.java

```diff
@@ -126,8 +136,13 @@ protected RpcConnection(Configuration conf, HashedWheelTimer timeoutTimer, Conne
       LOG.debug("Use " + authMethod + " authentication for service " + remoteId.serviceName
           + ", sasl=" + useSasl);
     }
+
     reloginMaxBackoff = conf.getInt("hbase.security.relogin.maxbackoff", 5000);
     this.remoteId = remoteId;
+
+    forceReloginEnabled = conf.getBoolean("hbase.security.force.relogin.enabled", false);
```

Comment:
Is there a reason this shouldn't default to 'true'? Otherwise we don't clear the cache and we should?

Reply:
Yeah, that's a good point. we can actually default it to true and since the default minimum time between is anyways 10 mins which matches with that of the time in UGI for non forceful logins